app/src/main/java/com/hammer/talkbbokki/presentation/topics/TopicListScreen.kt

```diff
@@ -160,29 +188,31 @@ fun SelectBtn(viewCnt: Int, onCardClicked: () -> Unit, modifier: Modifier) {
         shape = RoundedCornerShape(8.dp)
     ) {
         Text(
-            text = if (viewCnt >= 3) "광고 보고 뽑기" else "이 카드 뽑기",
-            style = TalkbbokkiTypography.button_large,
-            color = Color.White
+            text = if (isOpened)
+                stringResource(R.string.list_re_pick_btn)
+            else if (todayViewCnt >= 3)
+                stringResource(R.string.list_ad_pick_btn)
+            else
+                stringResource(R.string.list_pick_btn),
+            style = TalkbbokkiTypography.button_large, color = Color.White
         )
     }
 }
 
 @Composable
-fun CardItem(definiteIndex: Int, currentOffset: Float) {
+fun CardItem(definiteIndex: Int, currentOffset: Float, viewModel: TopicListViewModel) {
     val pageOffsetWithSign = definiteIndex - currentOffset
     val pageOffset = pageOffsetWithSign.absoluteValue
     Box(
         modifier = Modifier
             .width(dimensionResource(id = R.dimen.card_width))
-            .aspectRatio(0.7f)
+            .aspectRatio(0.65f)
             .background(Color.Transparent)
             .zIndex(5f - pageOffset)
             .graphicsLayer {
                 // 중간으로 올수록 크게 보임
                 lerp(
-                    start = 1f.dp,
-                    stop = 1.9.dp,
-                    fraction = 1f - pageOffset.coerceIn(0f, 1.3f)
+                    start = 1f.dp, stop = 1.9.dp, fraction = 1f - pageOffset.coerceIn(0f, 1.3f)
```

Comment:
🚫 **[ktlint]** <argument-list-wrapping> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Argument should be on a separate line (unless all arguments can fit a single line)